DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationController.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.controller('employerRegistrationController', function(
+    $scope,
+    stateService,
+    $location,
+    apiService
+  ) {
+    'ngInject';
+    'use strict';
+    $scope.showAllHelp = false
+    $scope.stateService = stateService;
+
+    $scope.navToLanding = function() {
+      $location.path('/dashboard');
+    };
+
+    $scope.showDetails = false;
+    $scope.toggleDetails = function ()  {
+      $scope.showDetails = !$scope.showDetails;
+      log.info($scope.showDetails);
```

Comment:
Is `log` globally defined?  I have a hazy memory that you'd put in a global logger that was then removed on production builds but I don't see anything in the webpack config that's reminding me of the details... 😬